stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/AttachmentsPickerDialogStyle.kt

```diff
@@ -0,0 +1,383 @@
+
+/**
+ * Style for [AttachmentSelectionDialogFragment].
+ *
+ * @param attachmentsPickerBackgroundColor The background color of the picker.
+ * @param allowAccessButtonTextStyle The text style used for all the button to the request required permissions.
+ * @param submitAttachmentsButtonIconDrawable The icon for the submit selected attachments button.
+ * @param attachmentTabToggleButtonStateList The color selector that will be applied to each tab button.
+ * @param mediaAttachmentsTabIconDrawable The icon for the image attachments tab.
+ * @param allowAccessToMediaButtonText The text to request required permissions on the image attachments tab.
+ * @param allowAccessToMediaIconDrawable The icon above the permissions text on the image attachments tab.
+ * @param videoLengthTextVisible If the video duration caption is displayed.
+ * @param videoLengthTextStyle The text style used for video duration caption in each tile.
+ * @param videoIconVisible If the video icon is displayed.
+ * @param videoIconDrawable The icon used for video files.
+ * @param mediaAttachmentNoMediaText The text that will be displayed if there are no media on the device.
+ * @param mediaAttachmentNoMediaTextStyle The text style for the the no media text.
+ * @param fileAttachmentsTabIconDrawable The icon for the file attachments tab.
+ * @param allowAccessToFilesButtonText The text to request required permissions on the file attachments tab.
+ * @param allowAccessToFilesIconDrawable The icon above the permissions text on the file attachments tab.
+ * @param recentFilesText The text above the file list on the file attachments tab.
+ * @param recentFilesTextStyle The text style of the recent files text on the file attachments tab.
+ * @param fileManagerIconDrawable The icon for a button to pick files from file manager.
+ * @param fileAttachmentsNoFilesText The text that will be displayed if there are no files on the device.
+ * @param fileAttachmentsNoFilesTextStyle The text style for the the no files text.
+ * @param fileAttachmentItemNameTextStyle The text style for the file name in a file attachment item.
+ * @param fileAttachmentItemSizeTextStyle The text style for the file size in a file attachment item.
+ * @param fileAttachmentItemCheckboxSelectedDrawable The drawable for the selected state of the checkbox in file attachment item.
+ * @param fileAttachmentItemCheckboxDeselectedDrawable The drawable for the deselected state of the checkbox in file attachment item.
+ * @param fileAttachmentItemCheckboxTextColor The color of the checkbox  in file attachment item.
+ * @param cameraAttachmentsTabIconDrawable The icon for the camera attachments tab.
+ * @param allowAccessToCameraButtonText The text to request required permissions on the camera attachments tab.
+ * @param allowAccessToCameraIconDrawable The icon above the permissions text on the camera attachments tab.
+ */
+@ExperimentalStreamChatApi
+public data class AttachmentsPickerDialogStyle(
```

Comment:
Trying to name things the same way in UI Components and Compose. For example, in Compose we have `AttachmentsPicker` component, hence `AttachmentsPickerDialogStyle` here.